src/levels/compaction.rs

```diff
@@ -298,9 +307,9 @@ impl Targets {
     }
 }
 
-pub fn get_key_range(tables: &[Table]) -> Option<KeyRange> {
+pub fn get_key_range(tables: &[Table]) -> KeyRange {
     if tables.is_empty() {
-        return None;
+        return KeyRange::default();
```

Comment:
One member in `KeyRange` is `Empty`.

Reply:
I mean why not remove the `Empty` item in `KeyRange`

Reply:
I think it is a better way to make `Empty` as one kind of `KeyRange`. Otherwise, we should use `Option<KeyRange>` as parameter instead, in which case we should check if it is `None` first and then do match with `KeyRange`, I just think it is unnecessary.